ql/src/test/results/clientpositive/llap/jdbc_filter_expand_row_operator.q.out

```diff
@@ -0,0 +1,41 @@
+PREHOOK: query: CREATE EXTERNAL TABLE book (id int, title varchar(100), author int)
+STORED BY 'org.apache.hive.storage.jdbc.JdbcStorageHandler'
+TBLPROPERTIES (
+    "hive.sql.database.type" = "POSTGRES",
+    "hive.sql.jdbc.driver" = "org.postgresql.Driver",
+    "hive.sql.jdbc.url" = "jdbc:postgresql://localhost:5432/qtestDB",
+    "hive.sql.dbcp.username" = "qtestuser",
+    "hive.sql.dbcp.password" = "qtestpassword",
+    "hive.sql.table" = "book")
+PREHOOK: type: CREATETABLE
+PREHOOK: Output: database:default
+PREHOOK: Output: default@book
+POSTHOOK: query: CREATE EXTERNAL TABLE book (id int, title varchar(100), author int)
+STORED BY 'org.apache.hive.storage.jdbc.JdbcStorageHandler'
+TBLPROPERTIES (
+    "hive.sql.database.type" = "POSTGRES",
+    "hive.sql.jdbc.driver" = "org.postgresql.Driver",
+    "hive.sql.jdbc.url" = "jdbc:postgresql://localhost:5432/qtestDB",
+    "hive.sql.dbcp.username" = "qtestuser",
+    "hive.sql.dbcp.password" = "qtestpassword",
+    "hive.sql.table" = "book")
+POSTHOOK: type: CREATETABLE
+POSTHOOK: Output: database:default
+POSTHOOK: Output: default@book
+PREHOOK: query: explain cbo
+select * from book 
+where id = 0 or (id = 1 and author = 11) or (id = 2 and author = 22)
+PREHOOK: type: QUERY
+PREHOOK: Input: default@book
+#### A masked pattern was here ####
+POSTHOOK: query: explain cbo
+select * from book 
+where id = 0 or (id = 1 and author = 11) or (id = 2 and author = 22)
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@book
+#### A masked pattern was here ####
+CBO PLAN:
+HiveJdbcConverter(convention=[JDBC.POSTGRES])
+  JdbcFilter(condition=[AND(OR(=($0, 0), AND(=($0, 1), =($2, 11)), AND(=($0, 2), =($2, 22))), IN($0, 0, 1, 2))])
```

Comment:
The `IN($0, 0, 1, 2)` seems kinda redundant any idea why it's there?

Reply:
True it's probably out of the scope of this PR so let's create a separate ticket to track it further.

I am suspecting that `IN(ROW($6, $8), ROW(1999, 12), ROW(2001, 1))` cannot be easily pushed in some JDBC databases cause they may not support the ROW operator. Nevertheless, I don't see why we should have both
`OR(=($0, 0), AND(=($0, 1), =($2, 11)), AND(=($0, 2), =($2, 22)))` and  `IN($0, 0, 1, 2)` especially for the JDBC storage handler where the underlying DB is gonna apply its own optimizations.

Reply:
No but for jdbc databases, we are expanding all the ROW operators. The issue is present for non-jdbc plans too. I will create a ticket for this 👍🏼

Reply:
I will create a new ticket for this after this PR is merged, so that I can reference the q.out file